src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/conversion.cpp

```diff
@@ -15,7 +15,7 @@ const std::vector<ov::test::utils::ConversionTypes> conversionOpTypes = {
     ov::test::utils::ConversionTypes::CONVERT_LIKE,
 };
 
-const std::vector<std::vector<ov::Shape>> shapes = {{{1, 2, 3, 4}}};
+const std::vector<std::vector<ov::Shape>> shapes = {{{1, 2, 3, 4}}, {{1, 1, 3, 5}}};
```

Comment:
Any reason to have additional shape here?

Reply:
It was added to cover tail test cases. As now the new added CPU plugin custom SLT covers tail cases, I just remove the revision here. Thanks Dmitry!